backend/sozisel/lib/sozisel_web/schema/resolvers/session_resolvers.ex

```diff
@@ -25,6 +25,11 @@ defmodule SoziselWeb.Schema.Resolvers.SessionResolvers do
       thumbnail =
         session
         |> Map.put(:password_required, session.entry_password != nil)
+        |> Map.put(:agenda_entries, Sessions.list_agenda_entries(session.session_template_id))
+        |> Map.put(
+          :estimated_time,
+          Sessions.get_template(session.session_template_id).estimated_time
```

Comment:
In one `Repo.preload` you could fetch all agenda entries and the emplate instead of doing 2 separates queries. Then when you get the preloaded data you can flat map this value